ruby-engine/Gemfile

```diff
@@ -5,7 +5,6 @@ source "https://rubygems.org"
 # Gemfile
 
 gem "ffi"
-gem "fiddle"
```

Comment:
This is the problem, FFI is an alternative, this was a stale dep that should have been removed